app/elements/app-user-menu/app-user-menu.html

```diff
@@ -161,42 +167,96 @@
                 </paper-menu>
             </div>
         </paper-menu-button>
+
+        <organization-add></organization-add>
     </template>
 </dom-module>
 <script type="text/javascript" src="../../bower_components/md5/build/md5.min.js"></script>
 <script>
-Polymer({
-    is: 'app-user-menu',
-    properties: {
-        email: {
-            type: String
+    Polymer({
+        is: 'app-user-menu',
+        properties: {
+            email: {
+                type: String
+            },
+            size: {
+                type: Number,
+                value: function() {
+                    return window.devicePixelRatio > 1.5 ? 80 : 40;
+                }
+            },
+            placeholder: {
+                type: String,
+                value: 'http://findicons.com/files/icons/1072/face_avatars/300/i04.png'
+            },
+            gravatar: {
+                type: String,
+                computed: '_computeGravatar(email, size, placeholder)'
+            },
+            placeholder: {
+                type: String,
+                value: 'mist.io/resources/images/sprite-images/user.png'
+            },
+            organization: {
+                type: Object,
+                value: function() {
+                    return ORGANIZATION;
+                }
+            },
+            orgs: {
+                type: Array,
+                value: function() {
+                    return ORGS;
+                }
+            },
+            orgsList: {
+                type: Array,
+                computed: '_computeOrgsList(orgs)'
+            },
+            canCreateOrg: {
+                type: Boolean,
+                value: function() {
+                    return CAN_CREATE_ORG;
+                }
+            },
+            isPersonalContext: {
```

Comment:
we're getting rid of the personal context altogether, so no reason to have it on polymist